frontend/src/Commons/ProgressBar/ProgressItem.styles.ts

```diff
@@ -0,0 +1,49 @@
+import styled from '@emotion/styled';
+import * as tokens from '@bcgov/design-tokens/js';
+import { IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import screenSizes from '@Constants/ScreenSize';
+
+type ProgressProps = {
+  favIcon?: IconDefinition;
+  active?: boolean;
+};
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+const StyledItem = styled.div<ProgressProps>`
+  border-radius: 50%;
+  background-color: ${(props) => (props.active === true ? tokens.supportBorderColorWarning : '#FFF')};
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 5px;
+  width: 46px;
+  height: 46px;
```

Comment:
And here I came here to bring up this shouldn't be `min-width`, yet you already fixed it. classic OpenShift not updating the image

Reply:
We're having more OpenShift issues?  They had an outage yesterday, including Actions.  Can't hurt to retry.